src/index.scss

```diff
@@ -1,29 +1,16 @@
-$feature-flags: (
-  ui-shell: true,
-  grid-columns-16: true,
-);
```

Comment:
These should not be removed - it is causing additional padding in your layout.

Reply:
@kellychurchill i have updated it. Please check.